.gitignore

```diff
@@ -19,3 +19,6 @@ dependencies/*
 
 # Ignore log files from OpenSim crashing
 *.log
+
+# Ignore OSX attribute files
```

Comment:
.DS_Store is some OS X based file that gets autogenerated by the OS X file system. Every time I make changes to the repo on an OSX machine; GIT recognizes .DS_Store as either an addition or modified file.